services/operatorsinfo/operatorsinfo_inmemory.go

```diff
@@ -189,3 +189,10 @@ func (ops *OperatorsInfoServiceInMemory) GetOperatorInfo(ctx context.Context, op
 		return resp.operatorInfo, resp.operatorExists
 	}
 }
+
+func (ops *OperatorsInfoServiceInMemory) updateSocketMapping(operatorId types.OperatorId, socket types.Socket) {
+	if socket == "" {
+		ops.logger.Warn("Received empty socket for operator", "operatorId", operatorId)
```

Comment:
add a return here since it's a no-op if socket is empty

Reply:
[0aefb65](https://github.com/Layr-Labs/eigensdk-go/pull/212/commits/0aefb653a83afe3506107486519b61a5f60e01d3)